CONTRIBUTING.md

```diff
@@ -87,47 +87,66 @@ The process of proposing a change to Talawa Admin can be summarized as:
    1. **_General_:** 
       1. We need to get to 100% test coverage for the app. We periodically increase the desired test coverage for our pull requests to meet this goal.
       1. Pull requests that don't meet the minimum test coverage levels will not be accepted. This may mean that you will have to create tests for code you did not write. You can decide which part of the code base needs additional tests if this happens to you.
-   1. **_Testing_:**
+   
+   2. **_Testing_:**
+
       1. Test using this set of commands:
-      ```
+
+         ```
+         npm install --legacy-peer-deps
+         npm run test --watchAll=false --coverage
+         ```
+      
+      2. Debug tests in browser 
+   
+         You can see the output of failing tests in broswer by running `jest-preview` package before running your tests
+
+         ```
          npm install --legacy-peer-deps
+         npm run jest-preview
          npm run test --watchAll=false --coverage
-      ```
-   1. **_Test Code Coverage_:**
+         ```
+      
+         You don't need to re-run the `npm run jest-preview` command each time, simply run the `npm run test` command if the Jest Preview server is already running in the background, it'll automatically detect any failing tests and show the preview at `http://localhost:3336` as shown in this screenshot -
+
+         ![Debugging Test Demo](./public/jest-preview.webp)
```

Comment:
Create a directory for images and put the file there.

Reply:
Other images like `REACT_SITE_KEY.webp` are directly being stored at the same level, should I move them too? Or that's for another issue?

Reply:
Yes, move them all